File: chapter_8/challenge1.c
// Challenge 1, chapter 8
// jeffreydiaz27@gmail.com

#include <stdio.h>

int main(int argc, const char* argv[])
{
	for (int i = 99; i >= 0; i -= 3) {
		printf("%d\n", i);
		if ((i % 5) == 0)
			printf("Found one!\n");
	}

	return 0;
}

File: chapter_16/challenge1.m
// Challenge 1, Chapter 16
// jeffreydiaz27@gmail.com

#import <Foundation/Foundation.h>

int main(int argc, const char* argv[])
{
	@autoreleasepool {
		NSString* sentence = @"This is a short sentence.";
		NSString* subStr = @"short";
		
		NSRange subStrRange = [sentence rangeOfString:subStr options:NSCaseInsensitiveSearch];
		
		if (subStrRange.location != NSNotFound) {
			NSLog(@"Substring found!\n");
			NSLog(@"%@\n", [sentence substringWithRange:subStrRange]);
		} else {
			NSLog(@"Substring not found.\n");
		}
	}
	return 0;
}
